framework/tlvf/AutoGenerated/include/tlvf/wfa_map/tlvClientCapabilityReport.h

```diff
@@ -22,6 +22,7 @@
 #include <tlvf/ClassList.h>
```

Comment:
This commit (5ac1d342585845243a90ea271f7b31458260abce : tlvf: update cliaent caps report TLV auto-generated files) has a typo on the headline.
cliaent ->client